javav2/example_code/acmpca/src/test/java/PCATests.java

```diff
@@ -1,89 +1,391 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import com.example.acmpca.CreateCertificateAuthority;
-import com.example.acmpca.DescribeCertificateAuthority;
-import com.example.acmpca.GetCertificateAuthorityCertificate;
-import com.example.acmpca.ListCertificateAuthorities;
-import com.example.acmpca.ListTags;
-
+import com.example.acmpca.*;
 import org.junit.jupiter.api.*;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.acmpca.AcmPcaClient;
+import software.amazon.awssdk.services.acmpca.model.*;
```

Comment:
nit: Avoid star imports. This is unit tests so I'm not gonna hold up the PR over this, but just be aware going forward